api-tests/features/dl-auth-source-checks/dl-cri-dropout.feature

```diff
@@ -0,0 +1,309 @@
+      When I submit a 'next' event
+      Then I get a 'fraud' CRI response
+      When I submit '<fraud-details>' details to the CRI stub
+      Then I get a 'page-ipv-success' page response
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P2' identity
+      And I have a dcmaw VC without 'drivingPermit' details
+
+      Examples:
+        | update-type             | dl-details                                       | passport-details                           | fraud-details                       |
+        | given-names-only        | kenneth-changed-given-name-driving-permit-valid  | kenneth-changed-given-name-passport-valid  | kenneth-changed-given-name-score-2  |
+        | family-name-only        | kenneth-changed-family-name-driving-permit-valid | kenneth-changed-family-name-passport-valid | kenneth-changed-family-name-score-2 |
+
+    Scenario Outline: Change of name and address journey - User backs out of DL CRI - Returns to DCMAW to use passport
+      When I submit a '<update-type>' event
+      Then I get a 'page-update-name' page response
+      When I submit a 'update-name' event
+      Then I get a 'dcmaw' CRI response
+      When I submit '<dl-details>' details to the CRI stub
+      Then I get a 'drivingLicence' CRI response
+      When I call the CRI stub with attributes and get an 'access_denied' OAuth error
+        | Attribute | Values          |
+        | context   | "check_details" |
+      Then I get a 'uk-driving-licence-details-not-correct' page response
+      When I submit a 'next' event
+      Then I get a 'dcmaw' CRI response
+      When I submit '<passport-details>' details to the CRI stub
+      Then I get a 'page-dcmaw-success' page response with context 'coiAddress'
+      When I submit a 'next' event
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-changed' details to the CRI stub
+      Then I get a 'fraud' CRI response
+      When I submit '<fraud-details>' details to the CRI stub
+      Then I get a 'page-ipv-success' page response
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P2' identity
+      And I have a dcmaw VC without 'drivingPermit' details
+
+      Examples:
+        | update-type             | dl-details                                       | passport-details                           | fraud-details                                   |
+        | given-names-and-address | kenneth-changed-given-name-driving-permit-valid  | kenneth-changed-given-name-passport-valid  | kenneth-changed-given-name-and-address-score-2  |
+        | family-name-and-address | kenneth-changed-family-name-driving-permit-valid | kenneth-changed-family-name-passport-valid | kenneth-changed-family-name-and-address-score-2 |
```

Comment:
I've mentioned to Chris, but I'm campaigning to combine these routes together (by performing the same COI check), which should halve a lot of these test cases 🤞

Reply:
oooo nice! Should I get rid of these tests then or leave in for the meantime?

Reply:
Whichever is easiest tbh - if you keep it in then you could make a note on https://govukverify.atlassian.net/browse/PYIC-7701, or just squash them pre-emptively.